community/cypher/cypher/src/main/java/org/neo4j/cypher/internal/codegen/ParameterConverter.java

```diff
@@ -365,37 +367,37 @@ public void writeDuration( long months, long days, long seconds, int nanos ) thr
     @Override
     public void writeDate( long epochDay ) throws RuntimeException
     {
-        writeValue( DateValue.epochDate( epochDay ) );
+        writeValue( LocalDate.ofEpochDay( epochDay ) );
     }
 
     @Override
     public void writeLocalTime( long nanoOfDay ) throws RuntimeException
     {
-        writeValue( LocalTimeValue.localTime( nanoOfDay ) );
+        writeValue( LocalTime.ofNanoOfDay( nanoOfDay ) );
     }
 
     @Override
     public void writeTime( long nanosOfDayUTC, int offsetSeconds ) throws RuntimeException
     {
-        writeValue( TimeValue.time( nanosOfDayUTC, ZoneOffset.ofTotalSeconds( offsetSeconds ) ) );
+        writeValue( OffsetTime.of( LocalTime.ofNanoOfDay( nanosOfDayUTC ), ZoneOffset.ofTotalSeconds( offsetSeconds ) ) );
```

Comment:
An `OffsetTime` is internally comprised of a `LocalTime` and a ZoneId. Therefore, this constructor comes more naturally. For `LocalDateTime` we cannot have the more natural choice (`LocalDate` and `LocalTime`, since `epochSecond` is part of path Date and Time. This splitting was chosen since it makes most sense for the PropertyStore (I think).